steam/client/__init__.py

```diff
@@ -51,6 +51,8 @@ class SteamClient(CMClient, BuiltinBase):
     _cm_servers_timestamp = None       # used to decide when to update CM list on disk
     _reconnect_backoff_c = 0
     current_jobid = 0
+    cell_id = 0
+    servers = {}
```

Comment:
Mutable types should not be initialized here as it will be shared across all instances of the class